app/scripts/directives/overview/mobileClientRow.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+(function () {
+  angular.module('openshiftConsole').component('mobileClientRow', {
+    controller: [
+      '$scope',
+      '$filter',
+      '$routeParams',
+      'ProjectsService',
+      'DataService',
+      'APIService',
+      'MobileClientsService',
+      'ListRowUtils',
+      'Navigate',
+      'AuthorizationService',
+      MobileAppRow,
+    ],
+    controllerAs: 'row',
+    bindings: {
+      apiObject: '<',
+      state: '<',
+      bindings: '<'
+    },
+    templateUrl: 'views/overview/_mobile-client-row.html'
+  });
+
+  function MobileAppRow($scope, $filter, $routeParams, ProjectsService, DataService, APIService, MobileClientsService, ListRowUtils, Navigate, AuthorizationService) {
+    var row = this;
+    row.installType = '';
+
+    _.extend(row, ListRowUtils.ui);
+
+    row.bundleDisplay = "bundle id: " + row.apiObject.spec.appIdentifier;
+    switch (row.apiObject.spec.clientType) {
+      case 'android':
+        row.installType = 'gradle';
+        row.bundleDisplay = "package name: " + row.apiObject.spec.appIdentifier;
+        break;
+      case 'iOS':
+        row.installType = 'cocoapods';
```

Comment:
The iOS and cordova apps now have appdentifiers as well. Should `row.bundleDisplay` be added here too?

Reply:
It has a default value at line 32